README.md

```diff
@@ -144,6 +146,8 @@ builds:
   - -X main.version={{.Env.VERSION}}
 - id: bar
   main: ./foobar/bar/main.go
+  env:
+  - CGO_ENABLED=1
```

Comment:
We don't tend to support cgo well, can we use another example here instead?

Reply:
Good point, changed to `GOCACHE`.